spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
- [ ] [OPTIONAL] After implementing the validations in the Models, kindly make sure to add more tests, you can include cases where the model is not validated e.g. when the name is not present the test should not be valid. I suggest adding similar test cases in all of your test models

```ruby
it 'The name should not be blank' do
  subject.name = ''
  expect(subject).to_not be_valid
end
```